dashboard/assets/cloud-pages.css

```
@@ -0,0 +1,394 @@
+  --pc-text: #020617;
+  --pc-text-2: #27272a;
+  --pc-sidebar-bg: #27272a;
+  --pc-pane-bg: #fdfdfd;
+  --pc-border: #e5e5e5;
+  --pc-teal-accent: #2ee59b;
+  --pc-sidebar-w: 264px;
+  --pc-sidebar-w-collapsed: 82px;
+}
+
+.pc-font {
+  font-family: "Inter", system-ui, -apple-system, "Segoe UI", Roboto,
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)